app/models/participant_outcome.rb

```diff
@@ -2,7 +2,7 @@ class ParticipantOutcome < ApplicationRecord
   belongs_to :declaration
   has_many :participant_outcome_api_requests
 
-  validates :ecf_id, uniqueness: true
+  validates :ecf_id, uniqueness: { case_sensitive: false }
```

Comment:
Why are we making all these case insensitive?

Reply:
Unique constraints are `case insensitive` or `case_sensitive false` by default so the DB won't allow adding a value for ecf_id = "583162EE-00B8-4AB9-A668-996E8EAAD40B" if "583162ee-00b8-4ab9-a668-996e8eaad40b" is already there. 

Shoulda matches in the other hand[ is `case sensitive true` by default](https://github.com/thoughtbot/shoulda-matchers/blob/main/lib/shoulda/matchers/active_record/validate_uniqueness_of_matcher.rb#L190), so we need to add a `.case_insensitive` to the matcher, so the test passes.

`it { is_expected.to validate_uniqueness_of(:ecf_id).case_insensitive.with_message("ECF ID must be unique") }`

As we had to add `case_insensitive` to the spec, I'm adding `{ case_sensitive: false }` into the model as well to keep it consistent with the rest.